include/ck/tensor_operation/gpu/device/impl/device_gemm_multiple_d_wmma_cshuffle.hpp

```diff
@@ -105,7 +105,7 @@ struct DeviceGemmMultipleD_Wmma_CShuffle : public DeviceGemmMultipleD<ALayout,
 #endif
         }();
 
-        if constexpr(GemmSpec == GemmSpecialization::MNPadding)
+        if constexpr(GemmSpec == GemmSpecialization::MNKPadding)
```

Comment:
This doesn't look right.

I don't see K is being packed

Reply:
A fix commit landing, it utilize the `MatrixPadder` struct to help padding work in wmma device op.